### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by Amschel** [apple2373/figure-separator#1]
Could you share the training code and also the net architecture?

Thank you!

**Comment by apple2373** [comment]
What are you trying to do?  

As I mentioned in the paper, training is not done by tensorflow, and it is purely for the deployment purpose.  Training is done by darknet. I didn't change any C code in darknet. 

The command I use is like, 
```
./darknet detector train <path to data file> <path to cfg file> <path to darknet19_448.conv.23> -gpus 0,1
```
I attach my .cfg file and .data file. 
[clef.data.txt](https://github.com/apple2373/figure-separator/files/1345319/clef.data.txt)
[yolo-imageclef-split-only-randomres.cfg.txt](https://github.com/apple2373/figure-separator/files/1345320/yolo-imageclef-split-only-randomres.cfg.txt)

You can check the network architecture from the cfg file.

**Comment by evilla-19** [comment]
I am trying to adapt your figure-separator to recognize entire panels instead of single images (i.e. I would need a composite microscopy image for example to stay together if it belongs to the same panel label, e.g. 'A', instead of getting boundaries for each individual channel/image). I am currently training on darknet with the settings you provided and my own training dataset with boundaries for panels as I would like the network to learn them. So far, so good I think. I would like to use the rest of your tool and wrappers so I would ideally like to also generate the .pb file using darkflow and use this with your tool instead of the model you provide. Do you describe the exact darkflow commands you used for deployment anywhere or would you maybe be able to share those?
Thanks!
